lorien/Misc/Parser.gd

```diff
@@ -0,0 +1,197 @@
+class_name Parser
```

Comment:
Can we find a better name (not as generic) for `Parser`? Maybe DSLParser, GrammarParser?

Reply:
DSLParser is nice, I think!